alibi_detect/cd/classifier.py

```diff
@@ -1,6 +1,10 @@
 import numpy as np
 from typing import Callable, Dict, Optional, Union
-from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow
+from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow, has_sklearn
+
+if has_sklearn:
+    from sklearn.base import ClassifierMixin
+    from alibi_detect.cd.sklearn.classifier import ClassifierDriftSklearn
```

Comment:
Note that since `scikit-learn` is a core dependency of `detect` this will always be true. That being said it's probably ok to leave this for consistency with `tensorflow` which is also currently a core dependency (but might not be in the future).

Reply:
I added for consistency with `tf` as you mentioned.